src/components/themed/ThemedScene.tsx

```diff
@@ -30,17 +33,24 @@ interface Props {
 
 export function ThemedScene(props: Props) {
   const {
+    branding,
     children,
     onBack,
     onSkip,
     paddingRem,
     title,
     noUnderline = false
   } = props
+  const { parentButton } = branding ?? {}
```

Comment:
Please put `branding = {}` in the props destructuring, instead of using `?? {}` here. Always avoid `??` if you can use destructuring instead, since that helps reduce the amount of undefined stuff floating around in the program.